test/integration/backward_compatible/parallel/topic/TopicTest.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const expect = require('chai').expect;
+const TestUtil = require('../../../../TestUtil');
+const RC = require('../../../RC');
+
+describe('TopicTest', function () {
+    let client;
+    let topic;
+
+    const testFactory = new TestUtil.TestFactory();
+
+    before(async function () {
+        const cluster = await testFactory.createClusterForParallelTests();
```

Comment:
We need to skip this test if the client version is not more than or equal to 5.3 because this test is a backward compatible test. See TestUtil.markClientVersionAtLeast method. Use "5.3" version.

Reply:
Okay, done.

Reply:
It is currently commented out. Changing it.